Examples/Tests/FluxInjection/inputs_3d

```diff
@@ -0,0 +1,68 @@
+# Maximum number of time steps
+max_step = 2
+
+# number of grid points
+amr.n_cell = 16 16 16
+
+# The lo and hi ends of grids are multipliers of blocking factor
+amr.blocking_factor = 8
+
+# Maximum allowable size of each subdomain in the problem domain;
+#    this is used to decompose the domain for parallel calculations.
+amr.max_grid_size = 16
+
+# Maximum level in hierarchy (for now must be 0, i.e., one level in total)
+amr.max_level = 0
+
+# Geometry
+geometry.dims = 3
+geometry.prob_lo     = -4 -4 -4
+geometry.prob_hi     =  4  4  4
+
+# Deactivate Maxwell solver
+algo.maxwell_solver = none
+warpx.const_dt = 7e-9
+
+# Boundary condition
+boundary.field_lo = periodic periodic periodic
+boundary.field_hi = periodic periodic periodic
+
+# particles
+particles.species_names = electron proton
+algo.particle_shape = 3
+
+electron.charge = -q_e
+electron.mass = m_e
+electron.injection_style = NFluxPerCell
+electron.num_particles_per_cell = 100
+electron.surface_flux_pos = -1.
+electron.flux_normal_axis = y
+electron.flux_direction = +1
+electron.profile = constant
+electron.density = 1.
+electron.momentum_distribution_type = gaussianflux
+electron.ux_th = 0.1
+electron.uy_th = 0.1
+electron.uy_m = 0.07
+electron.uz_th = 0.1
```

Comment:
A detail I noticed. This injects particles over the whole plane of the domain. Since the particles have a transverse velocity spread, it is possible that some particles are lost out the sides. If so, this would have a small effect on the comparison made in the analysis since the number of particles would be slightly too small.

Reply:
Agreed. However, in this particular case, the boundary condition is set to `periodic` ; therefore particles are in fact not lost.